tests/unet_classification_test.py

```diff
@@ -1,18 +1,31 @@
 import torch
-import matplotlib.pyplot as plt
-import seaborn as sns
-from torch.utils.data import DataLoader
 import torchmetrics
+import seaborn as sns
+import matplotlib.pyplot as plt
+from huggingface_hub.file_download import hf_hub_download
 
 from models.unet import Unet
-from ecg_segmentation_dataset import ECGDataset
-from train import preprocess_dataset, intersection_over_union
+from train import preprocess_dataset
+from train_binary_classification import intersection_over_union
 
 if __name__ == "__main__":
     # initializing model
-    ckpt = torch.load("checkpoints/run-2023-08-14-09-24.ckpt")
-    model = Unet(in_channels=2, out_channels=1, dim=32, dim_mults=(1, 2, 4), kernel_size=7, resnet_block_groups=4)
-    model.load_state_dict(ckpt["model"])
+    checkpoint = torch.load(
+        hf_hub_download(repo_id="JasiekKaczmarczyk/ecg-segmentation-unet", filename="classification-2023-08-14-09-24.ckpt")
```

Comment:
Can you also add code showing how to upload checkpoints to huggingface? :)

Reply:
I added it into train.py at the end.